app/renderer/components/styles/global.js

```diff
@@ -129,6 +129,7 @@ const globalStyles = {
     switchNubShadow: '1px 1px 5px -2px black',
     buttonShadow: '0px 1px 5px -1px rgba(0, 0, 0, 1.0)',
     dialogShadow: '0px 8px 22px 0px rgba(0, 0, 0, .5)',
+    flyoutDialogBoxShadow: '2px 2px 8px #3b3b3b',
```

Comment:
I don't think we should be using non-black hex colors for shadows. `rgba` handles this particular thing much better, IMHO.